src/aux-runtime/runtime/AuxLibrary.spec.ts

```diff
@@ -241,6 +241,7 @@ import {
 import { YjsPartitionImpl } from '@casual-simulation/aux-common/partitions';
 import { applyUpdate } from 'yjs';
 import { CasualOSError } from './CasualOSError';
+import { array } from '@hapi/joi';
```

Comment:
@TroyceGowdy This import is incorrect. `@hapi/joi` is a library that helps perform schema validation, not create JavaScript arrays. Always make sure you know what a library is for before you use an import from it.